doc/manipulating_images/data_preparation.rst

```diff
@@ -70,36 +70,35 @@ relevant for the research questions at hand.
 simplifying these "data folding" steps that often preceed the actual
 statistical analysis.
 
-On an advanced note,
-the underlying philosophy of these classes is similar to `scikit-learn
-<http://scikit-learn.org>`_\ 's
-transformers. First, objects are initialized with some parameters guiding
-the transformation (unrelated to the data). Then the fit() method
+On an advanced note, the underlying philosophy of these classes is similar
+to `scikit-learn <http://scikit-learn.org>`_\ 's transformers.
+First, objects are initialized with some parameters guiding
+the transformation (unrelated to the data). Then the `fit()` method
 should be called, possibly specifying some data-related
 information (such as number of images to process), to perform some
 initial computation (e.g., fitting a mask based on the data). Finally,
-transform() can be called, with the data as argument, to perform some
-computation on data themselves (e.g. extracting time series from images).
+`transform()` can be called, with the data as argument, to perform some
```

Comment:
Not exactly sure what single backquotes do in sphinx even after reading [this](http://www.sphinx-doc.org/en/stable/markup/inline.html). Did you mean to use double-backquotes instead?